jest-config.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+module.exports = (options = {}) => {
+    const {
+        transform = {},
+        moduleNameMapper = {},
+        setupFilesAfterEnv = [],
+        coverageThreshold = {},
+        boilerplate = false,
+        react = false,
+    } = options;
+
+    const config = {
+        transform: {
+            '\\.js$': require.resolve('babel-jest'),
+            '\\.(png|jpg|jpeg|gif|webp|ico)$': require.resolve('jest-file'),
+            '\\.(eot|ttf|woff|woff2|otf)$': require.resolve('jest-file'),
+            '\\.(mp3|flac|wav|aac|ogg|oga|mp4|m4a|webm|ogv)$': require.resolve('jest-file'),
+            '\\.(svg)$': require.resolve('jest-file'),
+            ...transform,
```

Comment:
I think it's best if we would use `...options.transform` here instead of having an object destructuring in the start since it doesn't add much. Only flags (like `react`) that are not direct configurations, should be extracted from `options`.

Reply:
Some of the properties being spread must have default values so the spreading doesn't break (i.e. `options.coverageThreshold` which must be object by default because of its `.global` cal, and `options.setupFilesAfterEnv` must be defaulted to an array), and destructuring handles. 

I don't think i should destructure just part of the object... do you think i should use another way to apply default values?